lib/ddtrace/configuration/settings.rb

```diff
@@ -245,6 +246,23 @@ def runtime_metrics(options = nil)
         o.lazy
       end
 
+      settings :test_mode do
```

Comment:
I am pretty confused on when a user would use this setting, and what environments (it seems like there's a list of vendors it supports?). But I assume this will be addressed in documentation at some point?

Reply:
Yeah, it has to be documented. Presumably you'd set this mode in your test suite if you want to tune the tracer to be more test friendly (e.g. keep 100% traces, produce traces to a test buffer, etc)

We've had users ask for a test mode for a long time in Ruby. https://github.com/DataDog/dd-trace-rb/pull/187

Reply:
kk. i think as long as we're clear this will be documented in the future this can be resolved.